packages/scripts/src/utils/optimisticGovernorPayload.ts

```diff
@@ -0,0 +1,39 @@
+import { Provider } from "@ethersproject/providers";
+import { ERC20Ethers } from "@uma/contracts-node";
+import { utils } from "ethers";
+import { getContractInstanceWithProvider } from "./contracts";
+
+interface ApprovalPayload {
+  approvalTokenAddress: string;
+  proposalPayload: string;
+  explanation: string;
+  approvalAmount: string;
+  recipient: string;
+}
+
+// Helper function to generate the payload for an approval transaction to be proposed through the OptimisticGovernor.
+// This uses TOKEN, AMOUNT and RECIPIENT from environment variables, or falls back to the provided parameters.
+export async function createApprovalPayload(
```

Comment:
So this suite of scripts is mainly meant to just test the OG on testnet, right? The approval transaction here is just meant as an example transaction that is hardcoded to give you something to push through, right?

Reply:
Gotcha. Makes sense!